doc/source/roadmap.rst

```diff
@@ -98,11 +98,8 @@ however; we need to make it faster and make it easier to compare performance of
 optimizers via plotting performance profiles.
 
 
-Linear programming enhancements
--------------------------------
-
-Recently all known issues with ``optimize.linprog`` have been solved.  Now we
-have many ideas for additional functionality (e.g. integer constraints, sparse
-matrix support, performance improvements), see gh-9269.
-
+Statistics enhancements
+-----------------------
 
+The `scipy.stats` enhancements listed in the `scipy-roadmap-detailed`_ are of
+particularly high importance to the project.
```

Comment:
Agree with putting this here. Would be good to keep this roadmap self-contained though. Can we pick the top prios and make them explicit here? How about:

- Improve fitting and behavior for extreme parameter values by implementing distribution-specific methods (`fit`, `pdf`, ...)
- Complete the set of hypothesis tests (add concrete ones here)
- Add confidence intervals for all statistical tests.